src/main/java/uk/gov/companieshouse/api/strikeoffobjections/common/ApiLogger.java

```diff
@@ -31,6 +31,10 @@ public void errorContext(String context, Exception e) {
         LOGGER.errorContext(context, e, null);
     }
 
+    public void errorContext(String context) {
+        LOGGER.errorContext(context, null, null);
```

Comment:
I'm not sure what the purpose of this method would be?
Context is the requestId so it would just be an error message with a request id with no info on what went on?
I would reserve error calls for when there is an exception, perhaps define and throw a custom exception and add that to the log call

Reply:
Removed this method